src/tr1/game/output.c

```diff
@@ -846,7 +797,7 @@ void Output_DrawRoom(const ROOM_MESH *const mesh)
 void Output_DrawShadow(
     const int16_t size, const BOUNDS_16 *const bounds, const ITEM *const item)
 {
-    g_ShadowInfo.vertex_count = g_Config.enable_round_shadow ? 32 : 8;
+    m_Shadow.vertex_count = g_Config.enable_round_shadow ? 32 : 8;
```

Comment:
Can we just allocate it on the stack here? Eg. just `SHADOW_INFO shadow = { .vertex_count = … };`